string robot_name
string[] joint_pose_list
string[] tcp_pose_list
